specification/compute/resource-manager/readme.python.md

```diff
@@ -52,167 +62,270 @@ batch:
   - tag: package-compute-2015-06
 ```
 
+```yaml $(multiapi) && $(track2)
+clear-output-folder: true
+package-top-folder: compute_preview
+batch:
+  - tag: package-2019-12-01-only
+  - tag: package-2019-11-01-only
+  - tag: package-2019-07-01-only
+  - tag: package-2019-04-01-only
+  - tag: package-2019-03-01-only
+  - tag: package-2018-10-01-only
+  - tag: package-2018-09-30-only
+  - tag: package-compute-only-2018-06
+  - tag: package-compute-2018-04
+  - tag: package-compute-only-2017-12
+  - tag: package-skus-2017-09
+  - tag: package-compute-2017-03
+  - tag: package-compute-2016-04-preview
+  - tag: package-compute-2016-03
+  - tag: package-compute-2015-06
+  - multiapiscript: true
+```
+
+``` yaml $(multiapiscript)
```

Comment:
this is for track1 right, why add a multiapi for track1 also

Reply:
track2 does it in different way than track1, this is a new way of doing it for track2